connection/cluster/ClusterDatabaseManager.java

```diff
@@ -59,21 +59,21 @@ public ClusterDatabaseManager(ClusterClient client) {
 
     @Override
     public boolean contains(String name) {
-        return failsafeTask(name, (stub, coreDbMgr) -> coreDbMgr.contains(name));
+        return failsafeTask(name, (stub, dbMgr) -> dbMgr.contains(name));
     }
 
     @Override
     public void create(String name) {
-        failsafeTask(name, (stub, coreDbMgr) -> {
-            coreDbMgr.create(name);
+        failsafeTask(name, (stub, dbMgr) -> {
+            dbMgr.create(name);
             return null;
         });
     }
 
     @Override
     public Database.Cluster get(String name) {
-        return failsafeTask(name, (stub, coreDbMgr) -> {
-            if (contains(name)) {
+        return failsafeTask(name, (stub, dbMgr) -> {
+            if (dbMgr.contains(name)) {
```

Comment:
call `dbMgr.contains` directly. The `get` function is already wrapped inside `failsafeTask`, so there's no need to call `contains(...)` which is also wrapped in `failsafeTask`.